### ISSUE TITLE
De-select edited notes with a long tap

### ISSUE THREAD

**Issue by vokinpirks** [git-moss/DrivenByMoss#334]
Hi, 
Push 2 user here (tho I guess this might be related to some other controllers as well).
When I'm in, let's say, the device mode (the display shows device parameters) and I long tap a step in a sequencer, the note edit mode activates where I can adjust the selected note's parameters. It would be great if when I'm done with note editing I could quit the note mode by just long tapping the step again and return back to the previously active mode. It just seems so logical and intuitive. Now I have to hold Select and tap a pad to de-select a note and the display shows 'Please select a note to edit' so I also have to manully switch to the mode where I was before.

I tried to do some hacking myself in [AbstractSequencerView](https://github.com/git-moss/DrivenByMoss/blob/4d5fcf72a6d08b93655cf52ce3523ca7d6c188f7/src/main/java/de/mossgrabers/framework/view/sequencer/AbstractSequencerView.java#L362)      

I changed it as follows

```java
        if (mode instanceof final INoteMode noteMode)
        {
            final List<NotePosition> editNotes = getEditNotes();
            if (editNotes.size() == 1 && editNotes.contains(notePosition)) {
                this.clearEditNotes();
                if (modeManager.isActive(Modes.NOTE)) {
                    modeManager.restore();
                }
            } else {
                if (addNote) {
                    noteMode.addNote(clip, notePosition);
                }
                else
                    noteMode.setNote (clip, notePosition);
                modeManager.setActive (Modes.NOTE);
            }
        }
```

The hack works with the regular note sequencer and drum sequencers, but unfortunately I had no success with the poly sequencer. It simply won't do anything. 
Also, I think it would be a great addition for everyone. 
In the video below you can see behaviour which I'm requesting for. This was recorded with the hacked version of a driver.

[see a video](https://photos.app.goo.gl/2Ffx2x7qWay7xRMLA)

Basically, this FR consists of two:
1) de-select selected note with a long tap
2) return back to the previous mode when editing is done (i.e. selection is clear)

**Comment by git-moss** [comment]
Yes, makes totally sense. Will add it to the wish list.
